Makefile

```diff
@@ -121,7 +121,7 @@ else
 	$(SED_I) "s/CROSS_BUILD_//g" $(DOCKERFILE)
 endif
 
-	$(DOCKER) build --no-cache --pull -t $(MULTI_ARCH_IMG):$(TAG) $(TEMP_DIR)/rootfs
+	$(DOCKER) build --no-cache -t $(MULTI_ARCH_IMG):$(TAG) $(TEMP_DIR)/rootfs
```

Comment:
I'll revert this, this is only so that I can use locally build base image using

```
BASEIMAGE=quay.io/kubernetes-ingress-controller/nginx-amd64:0.78 make dev-env
```

where I built `quay.io/kubernetes-ingress-controller/nginx-amd64:0.78` locally based on https://github.com/kubernetes/ingress-nginx/pull/3782